packages/dds/tree/src/simple-tree/arrayNode.ts

```diff
@@ -688,9 +691,6 @@ abstract class CustomArrayNodeBase<const T extends ImplicitAllowedTypes>
 		}
 
 		const sequenceField = getSequenceField(this);
-		// TODO: this is not valid since this is a value field schema, not a sequence one (which does not exist in the simple tree layer),
-		// but it works since cursorFromFieldData special cases arrays.
-		const simpleFieldSchema = normalizeFieldSchema(this.simpleSchema);
```

Comment:
skipping this needless conversion is a nice optimization, and also reduce code. Also allows deleting this out of date comment.